src/monitor.service.ts

```diff
@@ -32,42 +30,47 @@ export class MonitorService {
   }
 
   /**
-   * Create an observer for all endpoints
+   * Creates an observer for each endpoint
    */
   private startAllObservers() {
     this.observers = {};
     Object.values(this.endpoints).forEach((endpoint) => this.observers[endpoint.id] = this.startObserver(endpoint));
   }
 
   /**
-   * create a new CpuObserver with the specified data
-   * @param endpoint the given cpu endpoint
+   * Creates a new CpuObserver with the specified data
+   * 
+   * @param endpoint CPU endpoint that is going to be observed
+   * 
    * @returns a new cpu observer with the endpoint data
```

Comment:
mittlerweile ist mir klar geworden, dass meine fomulierung etwas ungenau war: mit 'blankline above tags' wollte ich eigentlich darauf hinaus, dass über dem gesamten block an tags _eine_ leerzeile sein muss, nicht das über jedem tag eine leerzeile sein muss :x
```suggestion
   * @param endpoint CPU endpoint that is going to be observed
   * @returns a new cpu observer with the endpoint data
```